hack/test-cmd.sh

```diff
@@ -74,6 +74,25 @@ function check-curl-proxy-code()
   echo "For address ${full_address}, got ${status} but wanted ${desired}"
   return 1
 }
+
+# TODO: remove this function when we do the retry inside the kubectl commands.
```

Comment:
Please make sure that there is an issue to track adding retries to kubectl, and mention it here.